README.md

```diff
@@ -1,32 +1,239 @@
 ### [README still WIP, not contribution friendly yet]
 
-### Monitoring Server Configuration
-In `prometheus/prometheus.yml`, add hosts emitting metrics to the targets under scrape config's node job.
+Nodies Monitoring is a customizable and extensible monitoring solution for monitoring host machine metrics, container metrics, node status and node logs. 
+
+## Table of content
+
+- [Installation](#installation)
+    - [Docker Dependencies](#docker-dependencies)
+    - [Nodies Monitoring](#nodies-monitoring)
+- [Setup](#setup)
+    - [.env](#env)
```

Comment:
Instead of asking to create a .env, can we include a .env.template?

Then you can have instructions include modify .env.template then do `cp .env.template .env`